CMakeLists.txt

```diff
@@ -164,13 +164,15 @@ set(${extension_name}_SOURCE_DIR "${CMAKE_BINARY_DIR}/${extension_name}")
 FetchContent_Populate(${extension_name}
  SOURCE_DIR     ${${extension_name}_SOURCE_DIR}
  GIT_REPOSITORY ${EP_GIT_PROTOCOL}://github.com/OpenwaterHealth/SlicerOpenLIFU.git
- GIT_TAG        0ea13d4fd39937059085f51fd14f60e6eca81397
+ GIT_TAG        5195604fdaf6562608a1a5cc05ce55887c33f0d5
```

Comment:
Update this after https://github.com/OpenwaterHealth/SlicerOpenLIFU/issues/3 is merged